webpack-2/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I'd love to but I'm not satisfying the peer dependency of `babel-loader` which hasn't upgraded to webpack@2, so npm is unhappy and doesn't allow me to create one :(